seed/challenges/08-coding-interview-questions-and-take-home-assignments/project-euler-problems.json

```diff
@@ -421,17 +421,129 @@
+        "  '40957953066405232632538044100059654939159879593635',",
+        "  '29746152185502371307642255121183693803580388584903',",
+        "  '41698116222072977186158236678424689157993532961922',",
+        "  '62467957194401269043877107275048102390895523597457',",
+        "  '23189706772547915061505504953922979530901129967519',",
+        "  '86188088225875314529584099251203829009407770775672',",
+        "  '11306739708304724483816533873502340845647058077308',",
+        "  '82959174767140363198008187129011875491310547126581',",
+        "  '97623331044818386269515456334926366572897563400500',",
+        "  '42846280183517070527831839425882145521227251250327',",
+        "  '55121603546981200581762165212827652751691296897789',",
+        "  '32238195734329339946437501907836945765883352399886',",
+        "  '75506164965184775180738168837861091527357929701337',",
+        "  '62177842752192623401942399639168044983993173312731',",
+        "  '32924185707147349566916674687634660915035914677504',",
+        "  '99518671430235219628894890102423325116913619626622',",
+        "  '73267460800591547471830798392868535206946944540724',",
+        "  '76841822524674417161514036427982273348055556214818',",
+        "  '97142617910342598647204516893989422179826088076852',",
+        "  '87783646182799346313767754307809363333018982642090',",
+        "  '10848802521674670883215120185883543223812876952786',",
+        "  '71329612474782464538636993009049310363619763878039',",
+        "  '62184073572399794223406235393808339651327408011116',",
+        "  '66627891981488087797941876876144230030984490851411',",
+        "  '60661826293682836764744779239180335110989069790714',",
+        "  '85786944089552990653640447425576083659976645795096',",
+        "  '66024396409905389607120198219976047599490197230297',",
+        "  '64913982680032973156037120041377903785566085089252',",
+        "  '16730939319872750275468906903707539413042652315011',",
+        "  '94809377245048795150954100921645863754710598436791',",
+        "  '78639167021187492431995700641917969777599028300699',",
+        "  '15368713711936614952811305876380278410754449733078',",
+        "  '40789923115535562561142322423255033685442488917353',",
+        "  '44889911501440648020369068063960672322193204149535',",
+        "  '41503128880339536053299340368006977710650566631954',",
+        "  '81234880673210146739058568557934581403627822703280',",
+        "  '82616570773948327592232845941706525094512325230608',",
+        "  '22918802058777319719839450180888072429661980811197',",
+        "  '77158542502016545090413245809786882778948721859617',",
+        "  '72107838435069186155435662884062257473692284509516',",
+        "  '20849603980134001723930671666823555245252804609722',",
+        "  '53503534226472524250874054075591789781264330331690'",
+        "];",
+        "",
+        "largeSum(testNums);",
+        "largeSum(fiftyDigitNums);"
```

Comment:
Maybe we should only call with the testNums whilst working on the problem.

Reply:
@Bouncey, sure thing! I'll make that change, too. I'll be sure to check my other PRs and do the same if they exhibit similar problems with lag.